### DISCUSSION TITLE
File not Found error in Dotplot

### DISCUSSION THREAD

**Discussion by u7068758** [cogent3/cogent3#1056] [Q&A] [answered]
Hi,
I want to perform Dot plot of two protein sequences. Can you guide me to the exact procedure? I've tried using the following command resulting in the "File not found error" as shown below.
![image](https://user-images.githubusercontent.com/107029444/172532776-76d4349a-5663-4908-9384-497f4fac8d25.png)

**Comment by GavinHuttley** [comment]
There are two errors in your code. The first is the path error.

File paths need to be absolute, or relative. I'm not sure if this will translate for Windows, but give it shot
```python
seqs = load_unaligned_seqs("~/Desktop/...
```
NOTE: `~/` means from home directory.

The second error is you need to specify the format. Please [read the cogent3 docs](https://cogent3.org/doc/cookbook/loading_sequences.html#specifying-the-file-format) on how to do that.

**Reply by u7068758** [reply]
Hi,
I've tried this way. Now, it is giving a Syntax error. I think I am not able to write the path correctly. Can you indicate the mistake?
![image](https://user-images.githubusercontent.com/107029444/172551137-d10f3ecc-6382-4180-b36d-67ca87be9a00.png)

**Reply by u7068758** [reply]
Yup, there was a mistake in the path. Now, after correction. I obtained the dot plot
![image](https://user-images.githubusercontent.com/107029444/172551911-9c8c5e85-fbcb-4a8a-93d1-dd5bc0f040ef.png)

**Reply by u7068758** [reply]
Can you guide me about the result interpretations? How can I best describe the results of Dot plot output in the thesis?
As per my understanding, the straight line indicates the matching amino acids and short parallel lines indicate repeats. Thus, we can say both the sequences are similar having tandem repeats at the end. At the top of output, it is written Matched>=13/20 what does this mean?
Also, Gap <=0 but according to alignment (clustal omega) there is one repeat missing in sequence on y-axis
![image](https://user-images.githubusercontent.com/107029444/172557244-da9c3a60-3efe-4b8b-a285-124c2fe3d19d.png)

**Reply by GavinHuttley** [reply, marked as answer]
That's a new discussion topic!!

Can you please move the above question into a separate topic. Thanks!
